Modules/xASL_module_Population.m

```diff
@@ -312,12 +293,26 @@
 		% Obtain a unique list of lesion names without the subject name
 		LesionUniqueROIList = unique(LesionROIList);
 
+		% Standard space analyzis in a specific ROI with no tissue restriction

```

Comment:
Perhaps we should create a new code section here for clarity, that this concerns lesions. And move all the lesion-specific stuff here. Something like:

%% ===============================================================
%% 7.b Lesion ....
%% ===============================================================

Reply:
Done